docs/academy/resources-quiz.json

```
@@ -0,0 +1,97 @@
+{
+    "quizTitle": "Test your knowledge!",
+    "nrOfQuestions": "6",
+    "questions": [
+      {
+        "question": "How are Bounded digital resources classified in terms of size?",
+        "questionType": "text",
+        "answerSelectionType": "single",
+        "answers": [
+          "They have infinite size.",
+          "They have a defined size and temporal limits.",
+          "They are limitless.",
+          "They are classified as structured and unstructured resources."
+        ],
+        "correctAnswer": "2",
+        "messageForCorrectAnswer": "Correct answer. Good job.",
+        "messageForIncorrectAnswer": "Incorrect answer. Please try again.",
+        "point": "1"
+      },
+      {
+        "question": "What is the main difference between Infrastructure Services and Digital Resource Processing Services ?",
+        "questionType": "text",
+        "answerSelectionType": "single",
+        "answers": [
+          "Infrastructure Services focus on managing complex processes, while Digital Resource Processing Services provide fundamental support for efficient resource sharing.",
+          "Infrastructure Services are responsible for resource identification, while Digital Resource Processing Services ensure the sovereignty of digital resources.",
+          "Infrastructure Services provide a foundation for efficient resource sharing, while Digital Resource Processing Services manage complex processes related to the transformation of digital resources.",
+          "Infrastructure Services specialize in resource storage, while Digital Resource Processing Services facilitate communication between network entities."
+        ],
+        "correctAnswer": "3",
+        "messageForCorrectAnswer": "Correct answer. Good job.",
+        "messageForIncorrectAnswer": "Incorrect answer. Please try again.",
+        "point": "1"
+      },
+      {
+        "question": "What do Verifiable Credentials (VCs) bring to the OKP4 ?",
+        "questionType": "text",
+        "answerSelectionType": "single",
+        "answers": [
+          "A method for storing resources directly on the blockchain.",
+          "A way to ensure the sovereignty of digital resources.",
+          "A mechanism for identifying resources without centralized control.",
+          "A means to describe specific properties of a resource"

```

I'd suggest adding a little more specificity to this mechanism:

```suggestion
          "A means to describe specific properties of a resource securely and in a verifiable manner"

```

what do you think?